backend/siarnaq/api/episodes/models.py

```diff
@@ -270,12 +283,84 @@ class Tournament(models.Model):
     def __str__(self):
         return self.name_short
 
-    def seed_by_scrimmage(self):
+    def get_potential_participants(self):
+        """Returns the list of participants that would be entered in this tournament,
+        if it were to start right now."""
+        # TODO incorporate real eligibility stuff
+        return (
+            Team.objects.with_active_submission()
+            .filter(episode=self.episode)
+            .all()
+            .order_by("-profile__rating__value")
+        )
+
+    def initialize(self):
         """
-        Seed the tournament with eligible teamsn in order of decreasing rating, and
-        populate the Challonge brackets.
+        Seed the tournament with eligible teams in order of decreasing rating,
+        populate the Challonge brackets, and create TournamentRounds.
         """
-        raise NotImplementedError
+
+        tour_name_public = f"{self.episode.name_long} {self.name_long}"
+        tour_name_private = tour_name_public + " (private)"
+
+        # For security by obfuscation,
+        # and to allow easy regeneration of bracket
+        key = random.randint(1000, 9999)
+        # Challonge does not allow hyphens in its IDs
+        # so substitute them just in case
+        tour_id_public = (f"{self.episode.name_short}_{self.name_short}_{key}").replace(
```

Comment:
Don't need to prefix with `self.episode.name_short` because `self.name_short` should already be unique.